src/components/jobs/status/StatusBadge.tsx

```
@@ -0,0 +1,70 @@
+        return "destructive";
+      case "accepted":
+        return "outline";
+      case "declined":
+        return "outline";
+      default:
+        return "default";
+    }
+  };
+
+  return (
+    <Badge variant={getStatusBadgeVariant() as any}>
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)